draft-ietf-ppm-dap.md

```diff
@@ -904,6 +905,8 @@ the following parameters associated with it:
 * `task_expiration`: The time up to which Clients are expected to upload to this
   task. The task is considered completed after this time. Aggregators MAY reject
   reports that have timestamps later than `task_expiration`.
+* `max_aggregation_job_size`: The maximum size, measured in number of reports,
```

Comment:
suggestion, feel free to ignore: "count" to align with "report_count" elsewhere?
```suggestion
* `max_reports_per_aggregation_job_count`: The maximum size, measured in number of reports,
```

Reply:
Went with `max_aggregation_job_report_count`, only because it's slightly shorter and (IMO) equally comprehensible.